stl/inc/ranges

```diff
@@ -8440,6 +8444,8 @@ namespace ranges {
     _EXPORT_STD template <class _Container, class... _Types>
         requires (!view<_Container>)
     _NODISCARD constexpr auto to(_Types&&... _Args) {
+        static_assert(!is_const_v<_Container>, "the target type must be cv-unqualified");
+        static_assert(!is_volatile_v<_Container>, "the target type must be cv-unqualified");

```

Comment:
```suggestion
        static_assert(!is_const_v<_Container>, "the target type must not be const. (N4928 26.5.7.2 [range.utility.conv.to])");
        static_assert(!is_volatile_v<_Container>, "the target type must not be volatile. (N4928 26.5.7.2 [range.utility.conv.to])");
```

Reply:
Ugh, please no subclause numbers - they change too frequently to be worth the trouble to update.

Reply:
What exactly do you mean by subclause numbers...? The `N4928`?

Reply:
no.

I wrote `26.5.7.2 [range.utility.conv.to]` and `26.5.7.3 [range.utility.conv.adaptors]` before.

So he meant 26.5.7.2 and 26.5.7.3